python/scannerpy/sampler.py

```diff
@@ -17,8 +17,26 @@ def _convert_collection(self, videos):
         else:
             return videos
 
-    def all(self, videos):
-        return self.strided(videos, 1)
+    def all(self, videos, item_size=1000, warmup_size=0):
+        sampler_args = self._db._metadata_types.AllSamplerArgs()
```

Comment:
All protobufs should be accessed via `db.protobufs.WhateverArgs` for consistency.